docs/source/custom_resolvers.rst

```diff
@@ -36,8 +36,8 @@ The example below creates a resolver that adds 10 to the given value.
     >>> c.key
     1000
 
-Custom resolvers support variadic argument lists in the form of a comma separated list of zero or more values.
-Whitespaces are stripped from both ends of each value ("foo,bar" is the same as "foo, bar ").
+Custom resolvers support variadic argument lists in the form of a comma-separated list of zero or more values.
+In a variadic argument list, whitespace is stripped from the ends of each value ("foo,bar" gives the same result as "foo, bar ").
 You can use literal commas and spaces anywhere by escaping (:code:`\,` and :code:`\ `), or
```

Comment:
At a glance, `:code:`\,`` looks like a surrounding double backticks. If so, I suggest we switch to double backtick for consistency with the rest of the doc.

Reply:
The doc build fails if I use double backticks for the one with the space. I think that the backslash is causing some escaping problem.